doc/man1/flux-mini.rst

```diff
@@ -365,12 +365,14 @@ OTHER OPTIONS
    otherwise VAL is interpreted as a string. If VAL is not set, then the
    default value is 1. See SHELL OPTIONS below.
 
-**--setattr=KEY=VAL**
+**--setattr=KEY[=VAL]**
    Set jobspec attribute. Keys may include periods to denote hierarchy.
-   VAL may be valid JSON (bare values, objects, or arrays), otherwise VAL
-   is interpreted as a string. If KEY starts with a ``^`` character, then
-   VAL is interpreted as a file, which must be valid JSON, to use as the
-   attribute value.
+   If KEY does not begin with ``system.``, ``user.``, or ``.``, then
+   KEY is set in ``attributes.system.`` by default.  VAL is optional and
```

Comment:
perhaps is confusing to say "does not begin with `system.` ...` then later say defaults to `attributes.system`.  Perhaps just say `system.` is assumed.

Reply:
You mean something like
```suggestion
   If KEY does not begin with ``system.``, ``user.``, or ``.``, then
   `system.` is assumed.  VAL is optional and
```

Yes, that sounds better.

Reply:
yeah, something like that.

Reply:
Ok, I made that change and forced a push. Thanks!